### PR TITLE
Add Epinio landing page

### PR DESCRIPTION
**Pull request by btat**
Fixes https://github.com/rancher/rancher-docs/issues/939

## Description

Adds a landing page for the Epinio integration.

### CODE REVIEW

**Comment by martyav** [comment]
@btat Do all the other landing page PRs also need the 2.8 milestone?

File: versioned_docs/version-2.8/integrations-in-rancher/epinio.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+Epinio lets developers go from application sources to URL in a single step. Epinio is an Application Development Platform. It deploys on Kubernetes and lets application developers and operators work together without conflict. 
+
+## Epinio with Rancher  
+
+Epinio's integration with Rancher lets developers quickly start using it without having to deal with the installation process or the configuration. You can install Epinio from the Apps. Currently the team is working to have Epinio available as a Rancher extension.
+
+## Epinio with Rancher Prime 
+
+On top of the specific support service, Rancher Prime customers of Epinio should expect better integration of Epinio with other Rancher projects such as:
```
Comment (martyav) [inline versioned_docs/version-2.8/integrations-in-rancher/epinio.md:17]:
```suggestion
Alongside Rancher-specific tech support, customers who combine a Rancher Prime subscription with Epinio can expect enhanced performance in the following areas: 
```

File: versioned_docs/version-2.8/integrations-in-rancher/epinio.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+---
+title: Application Development Engine with Epinio
+---
+
+<head>
+  <link rel="canonical" href="https://ranchermanager.docs.rancher.com/integrations-in-rancher/epinio"/>
+</head>
+
+Epinio lets developers go from application sources to URL in a single step. Epinio is an Application Development Platform. It deploys on Kubernetes and lets application developers and operators work together without conflict.
```
Comment (martyav) [inline versioned_docs/version-2.8/integrations-in-rancher/epinio.md:9]:
```suggestion
Epinio is a Kubernetes-based Application Development Platform. It helps operators and developers collaborate without conflict, and accelerates the development process. With Epinio, teams can move from application sources to a live URL in a single step.
```

File: versioned_docs/version-2.8/integrations-in-rancher/epinio.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+<head>
+  <link rel="canonical" href="https://ranchermanager.docs.rancher.com/integrations-in-rancher/epinio"/>
+</head>
+
+Epinio lets developers go from application sources to URL in a single step. Epinio is an Application Development Platform. It deploys on Kubernetes and lets application developers and operators work together without conflict. 
+
+## Epinio with Rancher  
+
+Epinio's integration with Rancher lets developers quickly start using it without having to deal with the installation process or the configuration. You can install Epinio from the Apps. Currently the team is working to have Epinio available as a Rancher extension.
```
Comment (martyav) [inline versioned_docs/version-2.8/integrations-in-rancher/epinio.md:13]:
```suggestion
Epinio's integration with Rancher gives developers a jump start, without having to deal with a lengthy installation process or configuration. You can install Epinio directly from the Rancher UI's Apps page. Currently the team is working to have Epinio available as a Rancher extension.
```

File: versioned_docs/version-2.8/integrations-in-rancher/epinio.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+## Epinio with Rancher Prime 
+
+On top of the specific support service, Rancher Prime customers of Epinio should expect better integration of Epinio with other Rancher projects such as: 
+
+- Better integration with Rancher authentication.  
+- Integrating Neuvector/Kubewarden with Epinio.
```
Comment (martyav) [inline versioned_docs/version-2.8/integrations-in-rancher/epinio.md:20]:
```suggestion
- Integration with Neuvector and Kubewarden.
```

File: versioned_docs/version-2.8/integrations-in-rancher/epinio.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+- Using a custom Chart template with the right annotations to integrate with monitoring for example.
```
Comment (martyav) [inline versioned_docs/version-2.8/integrations-in-rancher/epinio.md:21]:
```suggestion
- Using a custom Chart template with the right annotations to integrate with monitoring for example.
```

Not sure how to revise this to make it snappier..."Custom Helm chart templates for monitoring and other key tools"?

File: versioned_docs/version-2.8/integrations-in-rancher/epinio.md [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+- A better service marketplace.
```
Comment (martyav) [inline versioned_docs/version-2.8/integrations-in-rancher/epinio.md:22]:
```suggestion
- Improved service marketplace.
```

**Review by martyav — CHANGES_REQUESTED** [review]

**Comment by btat** [comment]
Per discussion with Guna, we're proceeding to merge so stakeholders are able to preview the initial design/update. We'll continue to improve/iterate and address any feedback post-merge.

**Pull request merged**